CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,170 @@
+# JSON Schema Organizational Code of Conduct
+
+Our (The JSON Schema Organization) Code of Conduct is a combination of
+the "Contributor Covenant Code of Conduct" 2.1 and the IETF BCP 54 "IETF Guidelines for Conduct" (RFC7154).
+
+# IETF Guidelines for Conduct
+
+Note: These guidelines are not provided in full in this document.
+Some extracts are provided.
+Please see the [RFC document](https://www.rfc-editor.org/rfc/rfc7154.html) for full details.
```

Comment:
That's only one of the documents - can you link to the other one also?

Reply:
Which ones? I'm not familiar.

Reply:
> "Contributor Covenant Code of Conduct" 2.1